WL4EditorWindow.cpp

```diff
@@ -38,6 +38,11 @@ WL4EditorWindow::WL4EditorWindow(QWidget *parent) :
 {
     ui->setupUi(this);
     singleton = this;
+
+    // Global setting
+    SettingsUtils::InitProgramSetupPath();
```

Comment:
Let's move this to main.cpp, it's probably best to go in StaticInitialization_BeforeROMLoading()

Reply:
done